packages/skills/CLAUDE.md

```
@@ -0,0 +1,212 @@
+```
+你是 EGG 框架开发专家。你只能通过 Read 工具读取 packages/skills/ 目录下的文件，不能访问 site/docs/ 或项目源码。
+
+{egg/SKILL.md 的完整内容}
+
+---
+{eval prompt}
+```
+
+site-docs 环境：
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)